### PR TITLE
Did one revision at README

### PR DESCRIPTION
**Pull request by whitead**
Also removed pypdf because it was confusing in the README

Direct link: https://github.com/Future-House/paper-qa/blob/readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,30 +4,52 @@
...
+
+PaperQA depends on some heroes that make our repo possible (in the order I remembered them):
+
+1. [Semantic Scholar](https://www.semanticscholar.org/)
+2. [Crossref](https://www.crossref.org/)
+3. [Unpaywall](https://unpaywall.org/)
+4. [Pydantic](https://docs.pydantic.dev/latest/)
+5. [Litellm](https://github.com/BerriAI/litellm)
+6. [pybtex](https://pybtex.org/)
+7. [pymupdf](https://pymupdf.readthedocs.io/en/latest/)
```
Comment (jamesbraza) [inline README.md:52]: Didn't we just remove this one?

**Comment by whitead** [comment]
Yeah this PR needs help. Will iterate again tonight

File: README.md (hunk 1 above)
Comment (whitead) [reply README.md:50]: confusing - no we removed `pypdf` not `pymupdf`

**Review by jamesbraza — APPROVED** [review]
Nice work

**Pull request merged**